packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -42,7 +40,7 @@ class _MatchesMethodCall extends Matcher {
   }
 }
 
-TextEditingController controller;
+TextEditingController? controller;
```

Comment:
Maybe we should just mark this as late and remove the `null` assignment in tearDown to remove all the `!` in the test? A disposed controller will already throw if used again, so I don't think there's much reason to null out the variable.